cmd/minikube/cmd/service.go

```diff
@@ -208,7 +208,7 @@ func startKicServiceTunnel(services service.URLs, configName, driverName string)
 			exit.Error(reason.DrvPortForward, "error getting ssh port", err)
 		}
 		sshPort := strconv.Itoa(port)
-		sshKey := filepath.Join(localpath.MiniPath(), "machines", configName, "id_rsa")
+		sshKey := filepath.Join(localpath.MiniPath(), "machines", configName, "id_ed25519")
```

Comment:
This looks fine, but why do we have to hard code the algorithm? why not use the default algorithm on the host? With this code we will have to change again when the default changes next time.